node/src/chain_spec.rs

```diff
@@ -520,7 +520,7 @@ fn amplitude_genesis(
 
 	let token_balances = balances
 		.iter()
-		.flat_map(|k| vec![(k.0.clone(), XCM(DOT), 1 << 60), (k.0.clone(), XCM(KSM), 1 << 60)])
+		.flat_map(|k| vec![(k.0.clone(), XCM(DOT), 0), (k.0.clone(), XCM(KSM), 0)])
 		.collect();
```

Comment:
Do we even have to iterate over it and assign empty balances? I guess we can just create an empty vec? 😅